configs/setup/system.json

```diff
@@ -13,5 +13,6 @@
 	"KubeVersion": "1.25.9-00",
 	"PmuToolsRepoUrl": "https://github.com/vhive-serverless/pmu-tools",
 	"ProtocVersion": "3.19.4",
-	"ProtocDownloadUrlTemplate": "https://github.com/protocolbuffers/protobuf/releases/download/v%s/protoc-%s-linux-x86_64.zip"
+	"ProtocDownloadUrlTemplate": "https://github.com/protocolbuffers/protobuf/releases/download/v%s/protoc-%s-linux-x86_64.zip",
+    "DebugMode": false
```

Comment:
fix the indent please

name it ```"LogVerbosity"```, should be 0 by default (but in CI better set to a high value). then you can directly use this value for setting verbosity, see below. people can increase verbosity by changing this value